webextensions/api/types.json

```diff
@@ -39,6 +39,50 @@
               "safari_ios": "mirror"
             }
           },
+          "clear": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/types/BrowserSetting/clear",
+              "support": {
+                "chrome": {
+                  "version_added": "≤58"
```

Comment:
Chrome doesn't support the Browser Settings API. What do the Chrome compat entries in this PR refer to?

Reply:
@dotproto I assume this is because, as mentioned in the documentation, this is provided as `ChromeSetting` in chrome. Adding a note to the BCD.